test/controllers/RegistrationProgressControllerSpec.scala

```diff
@@ -327,8 +327,8 @@ class RegistrationProgressControllerSpec extends AmlsSpec
 
             Seq(
               "businessmatching.registerservices.servicename.lbl.01",
-              "businessmatching.registerservices.servicename.lbl.02",
-              "businessmatching.registerservices.servicename.lbl.03"
+              "businessmatching.registerservices.servicename.lbl.03",
+              "businessmatching.registerservices.servicename.lbl.04"
```

Comment:
Thew numeric values in the sequence here seem to have gone from 10, 02, 03 => 01, 03, 04. (02 is missing). 

Is this intentional?

Reply:
Yes, the indices were updated due to the changes to message indices, but AMP hasn't been implemented elsewhere in the site yet